runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -2111,23 +2113,20 @@ MM_IncrementalGenerationalGC::exportStats(MM_EnvironmentVLHGC *env, MM_Collectio
 				if (region->containsObjects()) {
 					MM_MemoryPoolBumpPointer *memoryPool = (MM_MemoryPoolBumpPointer *)region->getMemoryPool();
 					Assert_MM_true(NULL != memoryPool);
-					/* for Eden region containing objects Allocation Age must be smaller then amount allocated since last PGC */
-					if (region->getAllocationAge() <= _allocatedSinceLastPGC) {
-						stats->_edenHeapSize += regionSize;
+					/* Eden region containing objects, Allocation Age must be smaller then amount allocated since last PGC,
+					 * more accurately, its logical age must be equal to zero */
+					if (0 == region->getLogicalAge()) {
```

Comment:
I prefer to use the old age condition (it's more generic), which might work better after reordering in https://github.com/eclipse/openj9/pull/11378